content/riak/ts/1.3.0/developing.md

```diff
@@ -16,15 +16,16 @@ canonical_link: "https://docs.basho.com/riak/ts/latest/developing"
 ---
 
 
-[erlang]: /riak/ts/1.3.0/developing/erlang
-[http]: /riak/ts/1.3.0/developing/http
-[java]: /riak/ts/1.3.0/developing/java
-[ruby]: /riak/ts/1.3.0/developing/ruby
-[python]: /riak/ts/1.3.0/developing/python
-[csharp]: /riak/ts/1.3.0/developing/csharp
-[nodejs]: /riak/ts/1.3.0/developing/nodejs
-[erlang]: /riak/ts/1.3.0/developing/erlang
-[php]: /riak/ts/1.3.0/developing/php
+[erlang]: /riak/ts/1.3.1/developing/erlang
+[go]: /riak/ts/1.3.1/developing/golang
+[http]: /riak/ts/1.3.1/developing/http
+[java]: /riak/ts/1.3.1/developing/java
+[ruby]: /riak/ts/1.3.1/developing/ruby
+[python]: /riak/ts/1.3.1/developing/python
+[csharp]: /riak/ts/1.3.1/developing/csharp
+[nodejs]: /riak/ts/1.3.1/developing/nodejs
+[erlang]: /riak/ts/1.3.1/developing/erlang
+[php]: /riak/ts/1.3.1/developing/php
```

Comment:
This is a 1.3.0 page linking to 1.3.1/ . Should these be pointing to 1.3.0/ or maybe to latest/ ?